.cirrus.yml

```
@@ -262,3 +262,19 @@ task:
         # TODO(jmagman): Lint macOS podspecs but skip any that fail library validation.
         - find . -name "*.podspec" | xargs grep -l "osx" | xargs rm
         - ./script/tool_runner.sh podspecs
+
+# Windows tasks
+task:
+  << : *FLUTTER_UPGRADE_TEMPLATE
+  gke_container:
+    dockerfile: .ci/DockerfileWindows
```

I guess you need to set platform. The default is `linux`

```suggestion
    dockerfile: .ci/DockerfileWindows
    platform: windows
```